stream.go

```diff
@@ -109,12 +109,10 @@ type ClientStream interface {
 //		1. Call Close on the ClientConn.
 //		2. Cancel the context provided.
 //		3. Call RecvMsg until a non-nil error is returned. For protobuf-generated clients:
-//			- Unidirectional clients should call CloseAndRecv (in the unidirectional case,
-//			  the server should only ever respond in the "closing" case). Note that
-//			  CloseSend will may not release all goroutines; CloseAndRecv guarantees
-//			  release of resources.
-//			- Bidirectional clients should call Close and then call Recv until a non-nil
-// 			  error is returned.
+//			- Unidirectional clients should call CloseAndRecv. Note that CloseSend may not
```

Comment:
Unidirectional"

Reply:
"client streaming RPCs" could include bidirectional, which does not have CloseAndRecv right?

Could you clarify how unidirectional is ambiguous? Are you referring to the fact that both clients and servers can unidirectionally stream? 

If so, how about:

- "For unidirectional RPCs, clients should ..."
- "For bidirectional RPCs, clients and servers should ..."

Reply:
Why not "For client-streaming RPCs..."?  We have historically called the two unidirectional streaming RPCs "client-streaming" and "server-streaming".